### ISSUE TITLE
Update cleaning notebook, derived datasets, and README with Season 6 Dryad data in GitHub Repo

### ISSUE THREAD

**Issue by MagicMilly** [genophenoenvo/terraref-datasets#91]
- [x] Will use functions from season 4 data cleaning to apply to season 6 Dryad data
- [x] Update and commit season 6 cleaning notebook to GitHub
- [x] Commit and upload derived season 6 datasets to GitHub and Google Drive
- [ ] Update README to include all updated metadata from incorporating Dryad data
- [x] Add latitude and longitude - maybe in a separate file with just plot sitenames, rather than adding to each dataset

Follow up to Issue #89 and #84

**Comment by MagicMilly** [comment]
- Updated Season 6 cleaning notebook can be found on [GitHub](https://github.com/MagicMilly/for-data-publication/tree/main/notebooks)
- Updated derived datasets for aboveground dry biomass and canopy height can be found on [GitHub](https://github.com/MagicMilly/for-data-publication/tree/main/data/processed) and Google [Drive](https://drive.google.com/drive/folders/1dXPER8sQD2wCfg_BQorDOdBNCoMwz_Bp?usp=sharing)
- Updated Season 4 code and derived datasets can also be found in the same GitHub locations and in this Google Drive [folder](https://drive.google.com/drive/folders/1gbB3Aa1F8ckyFk7qPr3LEiZCKPxk8DqJ?usp=sharing)

New code also includes some visualizations, but just distributions for now. 
<img width="391" alt="aboveground_dry_biomass_season_4" src="https://user-images.githubusercontent.com/34486154/88235873-9d6ec400-cc30-11ea-9bf3-83c93e622fbc.png">

**Comment by rbartelme** [comment]
@MagicMilly a few questions:

- How are the raw data imported into the Jupyter notebooks outside of CyVerse?

- Are the updated notebooks available inside the CyVerse VICE app?

- For the sake of uniformity do all the datasets contain some reference to `drought_treatment = FALSE`?

- Is there a tall format for each site/season that contains all the derived data? Same with the GDD by day in the weather data? I had previously modified the notebooks to export this myself, and could contribute to the code if necessary.

- Do all datasets need height blocking? Or was this a byproduct of the drought treatment in Season 4? (Maybe this is more of a question for @dlebauer)

**Comment by dlebauer** [comment]
yes, season 6 was also blocked by height https://terraref.ncsa.illinois.edu/bety/api/v1/experiments?name=MAC+Season+6:+Sorghum+BAP. This CSV file has the mappings from plot name to block for season 6 
[s6_blocks.csv.txt](https://github.com/genophenoenvo/terraref-datasets/files/4963090/s6_blocks.csv.txt)

**Comment by MagicMilly** [comment]
This personal [repo](https://github.com/MagicMilly/for-data-publication) now includes updated notebooks and data. Follow-up actions addressed in new ticket #93
